frontend/src/views/c-authorship.vue

```diff
@@ -339,34 +339,6 @@ export default {
       this.setInfoHash();
     },
 
-    getRepoProps(repo) {
```

Comment:
Can I ask whether it is safe to remove the getRepoProps method? Where will the logic handling authorDisplayName and filtering logic of author files go in this case

Reply:
@HCY123902 Thanks for catching the mistake here! I forgot about `authorDisplayName`. The logic is added back now.

As for the filtering logic, the `processFiles` function already does the filtering.

Reply:
Regarding the filtering logic for MergeGroup, the original practice is to directly use `repo.files`, while `isValidFile` only includes files with contributions from at least 1 known author. Can you verify that they are consistent? Alternatively, will this adjustment cause behavioral changes on the dashboard

Reply:
@HCY123902 They are consistent. `repo.files` contains all the files of the repo. This includes all the files that someone within the team has written to. It should work the same as before. If a file is not touched by anyone from the team, it should not show up in the first place.

Reply:
Thanks for the clarification